### PR TITLE
Select component

### PR DESCRIPTION
**Pull request by betoobernal**
## Summary

* Create filtrable Select Component.
* Find and request repositories bundle.
* Request new data when the scrolling comes down.

## Screenshots

select component

### CODE REVIEW

File: src/app/screens/AddOwner/layout.js [hunk 1]
```diff
@@ -1,15 +1,54 @@
...
 import AlertInfo from '~components/AlertInfo';
+import Select from 'app/components/Select';
+import { searchRepositories } from 'services/RepositoryService';
+import { parseArrayToObjectOfSelect } from 'utils/array';
+
+function AddOwner({ handleSubmit, onSubmit, reset, error, loading, submitSucceeded, submitFailed }) {
+  const [repository, setRepository] = useState(null);
+
+  const limit = 100;
+  const loadPageOptions = async (q, prevOptions, { page }) => {
```
Comment (FrankIglesias) [inline src/app/screens/AddOwner/layout.js]: what does `q` mean

File: src/app/screens/AddOwner/layout.js [hunk 2]
```diff
@@ -1,15 +1,54 @@
...
+    const {
+      data: { data }
+    } = await searchRepositories(page, q);
+
+    const options = data.length ? parseArrayToObjectOfSelect(data.map(e => e.split('/')[1])) : [];
```
Comment (FrankIglesias) [inline src/app/screens/AddOwner/layout.js]: e?

File: src/services/RepositoryService.js [hunk 3]
```diff
@@ -9,4 +9,5 @@ export const addOwnerToRepo = values =>
     codeowners: values.owners.split(',')
   });
 
-export const getRepositories = () => api.get('/repositories?limit=100&page=2');
+export const searchRepositories = (page = 1, query = '') =>
+  api.get(`/search/repositories?limit=100&page=${page}&query=${query}`);
```
Comment (FrankIglesias) [inline src/services/RepositoryService.js]:
```suggestion
  api.get('/search/repositories',{ limit: 100, page, query }); 
```

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: src/app/screens/AddOwner/layout.js (hunk 2 above)
Comment (betoobernal) [reply src/app/screens/AddOwner/layout.js]: Sorry! I made this code in two places at some moment, that's why the abstraction. But It wasn't like that and I forgot to remove it.

File: src/app/screens/AddOwner/layout.js (hunk 1 above)
Comment (betoobernal) [reply src/app/screens/AddOwner/layout.js]: D: that means "query"

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**